git_wrapper/branch.py

```diff
@@ -129,22 +130,24 @@ def rebase_to_hash(self, branch_name, hash_):
         )
 
         if self.git_repo.repo.is_dirty():
-            msg = ("Repository {0} is dirty. Please clean workspace "
-                   "before proceeding.".format(self.git_repo.repo.working_dir))
+            working_dir = self.git_repo.repo.working_dir
+            msg = (f"Repository {working_dir} is dirty. Please clean workspace "
+                   "before proceeding.")
             raise exceptions.DirtyRepositoryException(msg)
 
         # Checkout
         try:
             self.git_repo.git.checkout(branch_name)
         except git.GitCommandError as ex:
-            msg = "Could not checkout branch {name}. Error: {error}".format(name=branch_name, error=ex)
+            msg = f"Could not checkout branch {branch_name}. Error: {ex}"
             raise exceptions.CheckoutException(msg) from ex
 
         # Rebase
         try:
             self.git_repo.git.rebase(hash_)
         except git.GitCommandError as ex:
-            msg = "Could not rebase hash {hash_} onto branch {name}. Error: {error}".format(hash_=hash_, name=branch_name, error=ex)
+            msg = (f"Could not rebase hash {hash_} onto branch {branch_name}."
```

Comment:
Missing a space after the period.

Reply:
Those are good catches, thanks! While I was fixing those, I noticed a couple of our multi-line messages were not consistent regarding whether they broke to a new line for parens.  Since we seemed to have more that did NOT break, I went with that.  If you strongly prefer the other formatting of the parens, I can switch things that way instead, just let me know.

Reply:
No strong feelings either way! Probably why the discrepancy, heh. I like the change, thanks for the update.